pkg/group/group.go

```diff
@@ -86,26 +94,33 @@ func parseGroups(r io.Reader) (group map[string]Group, err error) {
 	return out, nil
 }
 
-func parseGroupLine(line string, group *Group) {
+func parseGroupLine(line string) (*Group, error) {
 	const (
 		NameIdx = iota
 		PassIdx
 		GidIdx
 		UsersIdx
 	)
+	var err error
 
 	if line == "" {
-		return
+		return nil, fmt.Errorf("cannot parse empty line")
 	}
 
 	splits := strings.Split(line, ":")
 	if len(splits) < 4 {
-		return
+		return nil, fmt.Errorf("invalid number of fields: %d", len(splits))
```

Comment:
Could you add that we expected at least 4 fields to the error message?

I was wondering if we should bail out if `len(splits) != 4`, but I guess that this is an unnecessary restriction here.